demo/admin/src/products/future/generated/ProductsGrid.tsx

```diff
@@ -120,7 +120,7 @@ export function ProductsGrid({ filter, toolbarAction, rowAction }: Props): React
             headerName: intl.formatMessage({ id: "product.type", defaultMessage: "Type" }),
             type: "singleSelect",
             valueOptions: [
-                { value: "Cap", label: intl.formatMessage({ id: "product.type.cap", defaultMessage: "Cap" }) },
+                { value: "Cap", label: intl.formatMessage({ id: "product.type.cap", defaultMessage: "great Cap" }) },
```

Comment:
@nsams am I understanding this correctly, that the column in the grid should now show "great Cap" instead of "Cap"? It works in the form but doesn't seem to work in the grid :thinking:

Reply:
Seems mui is not using valueOptions for rendering cell values.

Solving that is out of scope for this PR, but I guess it needs to be solved (else you always have english (from api) values)

Reply:
@jamesricky could you create a task and follow-up PR for this?

Reply:
> Solving that is out of scope for this PR

Isn't rendering custom values in the grid (and the form) exactly the point of this PR? 😅

Reply:
Created the task: SVK-334
Should I assign it to myself, @nsams or someone else?

FYI @thomasdax98

Reply:
> Isn't rendering custom values in the grid (and the form) exactly the point of this PR?

this PR allows setting custom labels for staticSelect values of a grid. Previously that was auto-generated from enum values (+ translation added).

The fact that valueOptions isn't used in the cell render function by mui, and that there is no comet code that does that, is a different issue that should be approached in a different PR